src/cargo/ops/cargo_package.rs

```diff
@@ -443,6 +444,14 @@ fn run_verify(ws: &Workspace<'_>, tar: &FileLock, opts: &PackageOpts<'_>) -> Car
     let pkg_fingerprint = hash_all(&dst)?;
     let ws = Workspace::ephemeral(new_pkg, config, None, true)?;
 
+    let rustc_args = if pkg.manifest().features().require(Feature::public_dependency()).is_ok() {
+        // FIXME: Turn this on at some point in the future
+        //Some(vec!["-D exported_private_dependencies".to_string()])
```

Comment:
Sorry for my ignorance. I don't know what this will do or why it is off now. Can you elaborate?

Reply:
This turns all `exported_private_dependencies` warnings into hard errors when publishing.